### DISCUSSION TITLE
GitHub Pages Not Deploying Website

### DISCUSSION THREAD

**Discussion by R4356th** [community/community#22164] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>The <a href="https://github.com/Glitchy-Scratch/gui/deployments/activity_log?environment=github-pages" rel="noopener nofollow ugc">GitHub Pages activity log</a> on my organisation’s repository says that there was a deployment an hour ago. However, if I visit the website at <a href="https://glitchy-scratch.js.org/gui/" rel="noopener nofollow ugc">https://glitchy-scratch.js.org/gui/</a>, I am shown an error 404. The site is being built from the gh-pages branch’s root which does have an index.html file. Could you please help me? Thank you very much.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Looks like that site is loading for me now. Did you make any further changes to fix this or did it just resolve on its own?</p>
<p>If it’s the latter it could just be it took some time to finish building and deploying and you then hit a 404 that ended up cached for a while.</p>

**Comment by R4356th** [comment]
<aside class="quote group-github-employees" data-username="thomasshaped" data-post="2" data-topic="151481">
<div class="title">
<div class="quote-controls"></div>
 thomasshaped:</div>
<blockquote>
<p>If it’s the latter it could just be it took some time to finish building and deploying and you then hit a 404 that ended up cached for a while.</p>
</blockquote>
</aside>
<p>Yes, it was the latter. It took about 50 minutes for the site to finally load.</p>

**Comment by otter-computer** [comment]
<p>Good to know, thanks for solving this! For the future, I believe a 404 will be cached if you visit your site before it’s finished fully deploying. You can bust the cache usually by adding a random URL param on the end of the URL—something like <code>?1234</code>.</p>
